src/util/nondet.cpp

```diff
@@ -7,6 +7,7 @@ Author: Diffblue Ltd.
 \*******************************************************************/
 
 #include "nondet.h"
+#include "allocate_objects.h"
```

Comment:
Nit pick:
````c++
#include "nondet.h"

#include "allocate_objects.h"
#include "arith_tools.h"
#include "c_types.h"
#include "fresh_symbol.h"
#include "symbol.h"
````

Reply:
Fixed the linebreaks in the same commit and removed the "util" parts in a new commit (same as the one for the forward declarations below).